.github/workflows/integration_test.yml

```diff
@@ -110,11 +110,11 @@ jobs:
         working-directory: frontend/appflowy_flutter
         run: |
           if [ "$RUNNER_OS" == "Linux" ]; then
-            flutter test integration_test -d Linux --coverage
+            flutter test integration_test/runner.dart -d Linux --coverage
```

Comment:
This PR adds a new integration test file. Although running `flutter test integration test [-args]` should work here, it appears that integration tests are unhappy when split between files. We should keep integration tests separated for readability and we can run them all from one test file `runner.dart`. See [this issue for context](https://github.com/flutter/flutter/issues/101031).